teuthology/task/install.py

```diff
@@ -577,6 +529,23 @@ def remove_sources(ctx, config):
             p.spawn(remove_fn, remote, project)
 
 
+def get_package_list(config):
+    debug = config.get('debuginfo', False)
+    project = config.get('project', 'ceph')
+    yaml_path = os.path.join(os.path.dirname(__file__), 'packages.yaml')
```

Comment:
I dont understand, I think we want to make it easy for install task to be configurable per branch, keeping this file here will only make it difficult again when it can easily be in qa-suites and its easier for one to configure it per ceph branch, Is there anything wrong with that approach?